src/app/mypage/record/components/BasicSold.css.ts

```diff
@@ -0,0 +1,66 @@
+import { style, globalStyle, styleVariants } from '@vanilla-extract/css';
+
+export const basicSold = style({
+  display: 'flex',
+  flexDirection: 'column',
+  listStyle: 'none',
+  maxHeight: '400px',
+  overflow: 'scroll',
+});
+
+export const list = style({
+  display: 'flex',
+  gap: '15px',
```

Comment:
```suggestion
  gap: '25px',
```
갭을 더 주는게 좋을 것 같아요. 판매 내역 부분에 정보가 많아서 간격이 좁아보이네요.